modules/compliance-operator-install.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance-operator/compliance-operator-rhacs.adoc
+:_module-type: PROCEDURE
+[id="compliance-operator-install_{context}"]
+= Installing Compliance Operator
```

Comment:
seems like this needs an article, but if not feel free to stet all of these:
```suggestion
= Installing the Compliance Operator
```

Reply:
That is a good question! I will ask around - I looked at existing documentation and it seems to do both, so I am confused. https://docs.openshift.com/container-platform/4.12/security/compliance_operator/compliance-operator-understanding.html

Reply:
(leaning towards adding "the")